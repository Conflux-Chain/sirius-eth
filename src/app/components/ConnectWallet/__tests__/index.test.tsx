import React from 'react';
import { render } from '@testing-library/react';
import { ConnectWallet } from '..';
import { MemoryRouter } from 'react-router';

describe('<ConnectWallet  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <MemoryRouter>
        <ConnectWallet />
      </MemoryRouter>,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
