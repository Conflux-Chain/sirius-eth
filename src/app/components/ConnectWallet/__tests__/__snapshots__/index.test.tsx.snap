// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ConnectWallet  /> should match snapshot 1`] = `
.c0 {
  height: 2.2857rem;
  background: #c9d5f1;
  border-radius: 1.1429rem;
  margin-right: 1.1429rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  justify-items: center;
  font-size: 1rem;
  font-weight: 500;
  color: #65709a;
  cursor: pointer;
}

.c0.pending {
  background: #fede1b;
  color: #ffffff;
}

.c0.pending .connect-wallet-button-left {
  background: #fede1b;
  color: #ffffff;
}

.c0:not(.pending):hover {
  background: #ffe872;
}

.c0:not(.pending):hover .connect-wallet-button-left {
  background: #ffe872;
}

.c0 .button-status-online {
  width: 0.5714rem;
  height: 0.5714rem;
  background: #7cd77b;
  border-radius: 0.2857rem;
  margin-right: 0.5714rem;
}

.c0 .button-status-pending {
  width: 0.8571rem;
  height: 0.8571rem;
  margin-right: 0.4286rem;
}

.c0 .balance {
  padding: 0 0.8571rem 0 0.5714rem;
}

.c0 .connect-wallet-button-left {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 2.2857rem;
  padding: 0 0.8571rem;
  background: #f5f6fa;
  border-radius: 3.5714rem;
  cursor: pointer;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

<span>
  <div
    class="c0 connect-wallet-button notConnected"
  >
    <span
      class="connect-wallet-button-left"
    >
      
      <span
        class="text"
      >
        Connect Wallet
      </span>
    </span>
  </div>
</span>
`;
